htdocs/expedition/card.php

```diff
@@ -1546,6 +1546,10 @@
 	{
 		print '<div class="tabsAction">';
 
+		$parameters = array();
```

Comment:
Add hooks may change behaviour of application and external module. Can you add it into dev branch ?